test/functional/Java8andUp/src/org/openj9/test/attachAPI/AttachApiTest.java

```diff
@@ -212,6 +217,17 @@ protected boolean checkLoadAgentOutput(String errOutput,
 		return outputLines;
 	}
 
+	protected List<String> runCommandAndLogOutput(List<String> commandLineOptions) throws IOException {
+		List<String> jpsOutput = runCommand(commandLineOptions);
+		StringWriter buff = new StringWriter();
+		PrintWriter buffWriter = new PrintWriter(buff);
+		jpsOutput.forEach(s -> {
+			buffWriter.println(s);
+		});
```

Comment:
In general, there should be a call to `buffWriter.flush()` here to be sure everything is written to `buff`.